### PR TITLE
[pipes] update databricks README

### PR DESCRIPTION
**Pull request by smackesey**

### CODE REVIEW

**Comment by smackesey** [comment]
Current dependencies on/for this PR:
* master
  * **PR #16909** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/16909" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/16909?utm_source=stack-comment).

**Review by yuhan — APPROVED** [review]

File: python_modules/libraries/dagster-databricks/README.md [hunk 1]
```diff
@@ -3,36 +3,38 @@
...
 import os
 from dagster import AssetExecutionContext, Definitions, asset
-from dagster_databricks import ExtDatabricks
+from dagster_databricks import PipesDatabricksClient
 from databricks.sdk import WorkspaceClient
 from databricks.sdk.service import jobs
 
 @asset
-def databricks_asset(context: AssetExecutionContext, ext: ExtDatabricks):
+def databricks_asset(context: AssetExecutionContext, pipes_client: PipesDatabricksClient):
```
Comment (schrockn) [inline python_modules/libraries/dagster-databricks/README.md:35]: Are we sure `pipes_client` is the naming convention we want? People will copy this.

`pipes`
`pipes_databricks_client`
`databricks_client`
`databricks_pipes`

I guess would be other options?

File: python_modules/libraries/dagster-databricks/README.md [hunk 2]
```diff
@@ -106,32 +108,33 @@ context.report_asset_materialization(
...
 
-### (2) `ext_protocol` context manager
+### (2) `open_pipes_session` context manager
 
-If you have existing code to launch/poll the job you do not want to change, or
-you just want more control than is permitted by `ExtDatabricks`, you can use
-`ext_protocol`. All that is necessary is that (1) your Databricks job be
-launched within the scope of the `ext_process` context manager; (2) your job is
+If you have existing code to launch/poll the job you do not want to change, you
+want to stream back results, or
```
Comment (schrockn) [inline python_modules/libraries/dagster-databricks/README.md:114]: very short line

**Review by schrockn — CHANGES_REQUESTED** [review]

File: python_modules/libraries/dagster-databricks/README.md (hunk 1 above)
Comment (smackesey) [reply python_modules/libraries/dagster-databricks/README.md:35]: IMO `pipes_client` is the best balance of clarity and concision.

**Review by schrockn — APPROVED** [review]

**Pull request merged**